Tests/FunctionalTests/PluginTests.swift

```diff
@@ -471,6 +471,22 @@ class PluginTests: XCTestCase {
         }
     }
 
+    func testLocalAndRemoteToolDependencies() throws {
+        // Only run the test if the environment in which we're running actually supports Swift concurrency (which the plugin APIs require).
+        try XCTSkipIf(!UserToolchain.default.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency")
+
+        try fixture(name: "Miscellaneous/Plugins/PluginUsingLocalAndRemoteTool") { path in
+            let (stdout, _) = try executeSwiftPackage(path.appending(component: "MyLibrary"), configuration: .Debug, extraArgs: ["plugin", "my-plugin"])
+            XCTAssert(stdout.contains("Linking RemoteTool"), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("Linking LocalTool"), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("Linking ImpliedLocalTool"), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("Build complete!"), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("A message from the remote tool."), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("A message from the local tool."), "stdout:\n\(stdout)")
+            XCTAssert(stdout.contains("A message from the implied local tool."), "stdout:\n\(stdout)")
```

Comment:
note I just merged the std/stderr cleanup PR so this may be different now?

Reply:
Yes, this will probably need to change.  Thanks for the heads-up.  Still not sure why this is failing on Linux — looks like the build log messages are not being emitted there — going to repo in Docker.

Reply:
the build output in such case is expected to go to stderr. this may also be why the linux build is failing

Reply:
Thanks, that's helpful.  Odd that it doesn't seem to happen on macOS.  In any case, I'll rebase on main and see what the right fix here is (ideally that also works unmodified on 5.6).  Technically speaking the build log output doesn't matter to this test — it's that we get the output from the plugin that matters, so perhaps those asserts aren't needed here.